src/frontend/src/lib/components/networks/NetworksSwitcher.svelte

```diff
@@ -26,18 +25,15 @@
 	$: testnets = $testnetsStore?.enabled ?? false;
 </script>
 
-<ButtonHero
+<ButtonSwitcher
 	bind:button
 	on:click={() => (visible = true)}
 	ariaLabel={$i18n.networks.title}
 	{disabled}
-	imgSrc={$selectedNetwork?.icon ?? chainFusion}
-	imgAlt={replacePlaceholders($i18n.core.alt.logo, {
-		$name: $selectedNetwork?.name ?? $i18n.networks.chain_fusion
-	})}>{$selectedNetwork?.name ?? $i18n.networks.chain_fusion} <IconChevronDown /></ButtonHero
+	>{$selectedNetwork?.name ?? $i18n.networks.chain_fusion} <IconChevronDown /></ButtonSwitcher
 >
 
-<Popover bind:visible anchor={button} direction="rtl">
+<Popover bind:visible anchor={button} direction="ltr">
```

Comment:
`ltr` is the default maybe?

Reply:
yes, you are right, just checked. I will remove it